db/post_migrate/20220325064954_make_some_bookmark_columns_nullable.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class MakeSomeBookmarkColumnsNullable < ActiveRecord::Migration[6.1]
+  def change
+    change_column_null :bookmarks, :post_id, true
```

Comment:
What do you think about using a constraint here instead? Something like if `bookmarkable_type` is `'Post'`, `post_id` is not nullable. I don't think it should be a blanket null-able as it has the potential to introduce bugs while we transition into the polymorphic assocaition.

Reply:
Yeah I can add one but it will be more like `if bookmarkable_type IS NOT NULL THEN bookmarkable_id NOT NULL ELSE post_id NOT NULL`

Reply:
Sounds good to me :+1: